h2d/Flow.hx

```diff
@@ -845,4 +873,4 @@ class Flow extends Object {
 	public dynamic function onAfterReflow() {
 	}
 

```

Comment:
I think it would be necessary to fully document the class before merge, as the style/vocabulary needs to be consistent.